guides/pages.yml

```
@@ -11,8 +11,8 @@
       url: 'index'
     - title: 'CLI commands'
       url: 'cli-commands'
-    - title: 'Folder Layout'
-      url: 'folder-layout'
+    - title: 'Folder layout'
+      url: 'folder-layout-and-naming-conventions'
```

Can we revert the URL change?

```suggestion
      url: 'folder-layout'
```